build/cocos2d_tests.xcodeproj/project.pbxproj

```diff
@@ -8372,7 +8375,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = H2S6TV5WJW;
```

Comment:
I think, you should remove your development from the project.

Reply:
Fixed, thanks :)